infrastructure/modules/service-bus/output.tf

```diff
@@ -1,7 +1,13 @@
 output "queues_default_primary_connection_string" {
-  value = ["${azurerm_servicebus_queue_authorization_rule.service_bus_queue_auth_rule.*.primary_connection_string}"]
+  value = tomap({
```

Comment:
also, because we are generating the resources using `for_each` now, the resources are not a list, but rather a map and so we need to iterate over them map to produce outputs